rs/cli/src/main.rs

```diff
@@ -12,6 +12,26 @@ struct CargoCommands {
 enum SailsCommands {
     #[command(name = "sails", subcommand)]
     Sails(Commands),
+    #[command(name = "sails-js", subcommand)]
+    SailsJs(JsCommands),
+}
+
+#[derive(Subcommand)]
+enum JsCommands {
+    #[command(name = "generate-client")]
+    GenerateClient {
```

Comment:
it is better to make `js-client` under sails, like
```
cargo sails js-client service.idl
```

Reply:
+1